src/main/resources/view/ReminderListCard.fxml

```diff
@@ -27,11 +27,11 @@
         <Label fx:id="message" text="\$first" wrapText="true" styleClass="cell_big_label"/>
       </HBox>
       <HBox spacing="5" alignment="CENTER_LEFT">
-        <FontIcon fx:id="calendarIcon" iconLiteral="far-calendar-alt" iconColor="white" iconSize="16"/>
+        <FontIcon fx:id="calendarIcon" iconLiteral="far-calendar-alt" styleClass="big-icon"/>
```

Comment:
I think this sets the icon to be permanently white even in the dark theme. By right the icon should be green when the reminder is complete, and red when the reminder is overdue.

In the light theme, the text doesn't reflect the colours as well.

Reply:
I'll try to fix it asap!